packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -214,6 +173,32 @@ function computeInterval(): number {
   // every day
   return 60 * 60 * 24 * SECOND;
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+let statusColumn = new Column<ImageInfoUI>('Status', 'center', '70px', ImageColumnStatus);
+statusColumn.setComparator((a, b) => {
+  let au: boolean = a.inUse;
+  let bu: boolean = b.inUse;
+  return au === bu ? 0 : au ? -1 : 1;
+});
+
+let nameColumn = new Column<ImageInfoUI>('Name', 'left', '3fr', ImageColumnName);
+nameColumn.setComparator((a, b) => a.name.localeCompare(b.name));
+
+let ageColumn = new Column<ImageInfoUI>('Age', 'left', '1fr', ImageColumnAge);
+ageColumn.setComparator((a, b) => a.name.localeCompare(b.name));
+
+const columns: Column<ImageInfoUI>[] = [
+  statusColumn,
+  nameColumn,
+  ageColumn,
+  new Column<ImageInfoUI>('Size', 'right', '1fr', ImageColumnSize),
+  new Column<ImageInfoUI>('Actions', 'right', '150px', ImageColumnActions),
+];
+
+const tableHelper = new TableHelper<ImageInfoUI>('image', image => image.inUse, 'Image is used by a container');
```

Comment:
I went with a Row class instead of TableHelper in the second latest commit, and moved the 'kind' as a standalone property. The Row is only for setting selected/disabled text for now. I debated whether selection should be a completely standalone thing, but I think using Row will lead to simple use long term as we add other options.